tests/worker/test_activity.py

```diff
@@ -1351,3 +1352,47 @@ def assert_activity_application_error(
     ret = assert_activity_error(err)
     assert isinstance(ret, ApplicationError)
     return ret
+
+
+@activity.defn
+async def activity_with_retry_delay():
+    raise ApplicationError(
+        ActivitiesWithRetryDelayWorkflow.error_message,
+        next_retry_delay=ActivitiesWithRetryDelayWorkflow.next_retry_delay,
+    )
+
+
+@workflow.defn
+class ActivitiesWithRetryDelayWorkflow:
```

Comment:
This test file was built for activities only against the kitchen sink workflow before the workflow worker was created in Python. I am not sure it is safe for use in a sandbox like `test_workflow.py` is and that's probably why CI is failing. I think this test should move into that file that is kept safe for sandbox use.